implementations/c/include/ockam/codec.h

```diff
@@ -156,6 +158,8 @@ uint8_t* decode_endpoint(uint8_t* encoded, CodecEndpointType* type, uint8_t* end
 uint8_t* encode_key_agreement(uint8_t* encoded, codec_payload_t* kt_payload);
 uint8_t* decode_key_agreement(uint8_t* encoded, codec_payload_t* kt_payload);
 uint8_t* encode_ockam_wire(uint8_t* p_encoded);
-uint8_t* decode_ockam_wire(uint8_t* p_encoded);
+uint8_t* decode_ockam_wire(uint8_t* p_encoded, uint16_t* p_version);
```

Comment:
I don't think it's important in this case, but in the future I think it would be good to create separate PRs for such things.